src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterSynchronized.java

```diff
@@ -0,0 +1,53 @@
+/*
+NoWhitespaceAfter
+allowLineBreaks = (default)true
+tokens = LITERAL_SYNCHRONIZED
+
+
+*/
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nowhitespaceafter;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+public class InputNoWhitespaceAfterSynchronized<T> {
+
+    private final Object lock = new Object();
+
+    void m1() {
+        synchronized (lock) {} // violation
```

Comment:
A bunch of variations on this can already be found in https://github.com/checkstyle/checkstyle/blob/edcc686e3099db8ccffb70b03cd5dcc0c862177b/src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterTestSynchronized.java#L11